Competition2018/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -25,6 +32,62 @@ public void setupShiftGearCommand(
 			OperatorInterface oi,
 			ToggleGearCommand shiftGear
 			) {
-		oi.gamepad.getifAvailable(4).whenPressed(shiftGear);
+		oi.driverGamepad.getifAvailable(6).whenPressed(shiftGear);
 	}
+	
+	@Inject
+	public void setupLeanCommands(
+			OperatorInterface oi,
+			LeanLeftCommand left,
+			LeanRightCommand right
+			) {
+		oi.operatorGamepad.getAnalogIfAvailable(new AnalogHIDDescription(0,0,.45)).whileActive(left);
+		oi.operatorGamepad.getAnalogIfAvailable(new AnalogHIDDescription(0,.55,1.0)).whileActive(right);
+	}
+	
+	@Inject
+	public void setupGripperCommands(
+			OperatorInterface oi,
+			GripperDeployDownCommand down,
+			GripperDeployUpCommand up,
+			GripperEjectCommand eject,
+			GripperIntakeCommand intake
+			) {
+		oi.operatorGamepad.getifAvailable(3).whenPressed(up);
+		oi.operatorGamepad.getifAvailable(2).whenPressed(down);
+		oi.operatorGamepad.getifAvailable(4).whenPressed(eject);
+		oi.operatorGamepad.getifAvailable(1).whenPressed(intake);
+	}
+	
+	@Inject
+	public void setupElevatorCommands(
+			OperatorInterface oi,
+			LowerCommand lower,
+			RiseCommand rise,
+			MoveToMaxHeightCommand max,
+			MoveToMinHeightCommand min
+			) {
+		oi.operatorGamepad.getAnalogIfAvailable( new AnalogHIDDescription(3,.8,1.0)).whileActive(max);
```

Comment:
Let's remove the max/min commands for now. Too dangerous until they have been tested.